### PR TITLE
Add Yale Module Clusters

### PR DESCRIPTION
**Pull request by danieledwardgeorgehitchcock**
enhances work done in Koenkk/zigbee2mqtt#4778

### CODE REVIEW

File: src/zcl/definition/cluster.ts [hunk 1]
```diff
@@ -5166,7 +5166,262 @@ const Cluster: {
             },
         },
         commandsResponse: {},
-    },    
+    },
+    manuSpecificAssaDoorLock: {
+        ID: 0xFC00,
```
Comment (Koenkk) [inline src/zcl/definition/cluster.ts:5171]: Could we add a manufacturer code here like; https://github.com/Koenkk/zigbee-herdsman/blob/064662f37da1c0538e52e09f80bc3f724eb693a9/src/zcl/definition/cluster.ts#L4141

Comment (danieledwardgeorgehitchcock) [reply src/zcl/definition/cluster.ts:5171]: I looked for one in the decrypted driver and sniffs but couldn't find one. Will dig a bit deeper

Comment (danieledwardgeorgehitchcock) [reply src/zcl/definition/cluster.ts:5171]: Just double checked the sniff, and there isn't any `manufacturerCode` that is sent along with the packets..?

image

Comment (Koenkk) [reply src/zcl/definition/cluster.ts:5171]: Thanks for checking!

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**